libraries/AP_ESC_Telem/AP_ESC_Telem_Backend.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <AP_HAL/AP_HAL.h>
+
+#ifndef HAL_WITH_ESC_TELEM
+#define HAL_WITH_ESC_TELEM 1
+#endif
+
+#if HAL_WITH_ESC_TELEM
+
+class AP_ESC_Telem;
+
+class AP_ESC_Telem_Backend {
+public:
+
+    struct TelemetryData {
+        int16_t  temperature_deg;   // degrees C, negative values allowed
```

Comment:
Shouldn't this one be in centi-degrees instead? [-655.36 ... +655.35] seams ok for most applications in this solar system :)

Reply:
Can be - I don't mind